schunk_gripper_dummy/schunk_gripper_dummy/config/data.json

```diff
@@ -57,7 +57,7 @@
         "40000000"
     ],
     "0x0588": [
-        "42A60000"
+        "0000E642"
```

Comment:
@PrajwalMangaluru I think you switched the byte `A6` to `E6`. Was that intentional?